_attributes/common-attributes.adoc

```diff
@@ -335,3 +335,5 @@ endif::openshift-origin[]
 // Hosted control planes related attributes
 :hcp-capital: Hosted control planes
 :hcp: hosted control planes
+:mce: multicluster engine for Kubernetes Operator
```

Comment:
I notice that you may want to update this in the topic_map.yml as well because in preview the operator is lowercase. 
![image](https://github.com/openshift/openshift-docs/assets/77501157/b02bd596-9099-4fc3-9025-7da122bc6268)